Deliverables/D4 documentation.md

```diff
@@ -0,0 +1,29 @@
+Alexander Craig
+**Proof of Concept**
+![1](https://user-images.githubusercontent.com/46610322/112400415-ba3b2a00-8cde-11eb-8e1c-fc3457bb9cca.jpg)
+![2](https://user-images.githubusercontent.com/46610322/112400416-bad3c080-8cde-11eb-9735-630424dec14c.jpg)
+![3](https://user-images.githubusercontent.com/46610322/112400418-bad3c080-8cde-11eb-9afa-fdc943cfcc84.jpg)
+![4](https://user-images.githubusercontent.com/46610322/112400419-bad3c080-8cde-11eb-828a-168c4a09c04e.jpg)
+![5](https://user-images.githubusercontent.com/46610322/112400420-bb6c5700-8cde-11eb-8f14-e3790b4d6ee2.jpg)
+![6](https://user-images.githubusercontent.com/46610322/112400421-bb6c5700-8cde-11eb-90f4-e137b1d218fd.jpg)
+![7](https://user-images.githubusercontent.com/46610322/112400423-bb6c5700-8cde-11eb-8737-84718ddb2074.jpg)
+![8](https://user-images.githubusercontent.com/46610322/112400410-b9a29380-8cde-11eb-9a54-ce48dbb28f53.jpg)
+![9](https://user-images.githubusercontent.com/46610322/112400411-b9a29380-8cde-11eb-8eef-781f010fadaf.jpg)
+![10](https://user-images.githubusercontent.com/46610322/112400412-ba3b2a00-8cde-11eb-9a21-925611dfe070.jpg)

```

Comment:
A one-sentence description per image is required in D4. This help us understand why each specific piece is part of your proof of concept.